extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -182,13 +191,12 @@ public function content( $params ){
 			}
 		}
 
-		$this->thumbWidth = self::IMG_WIDTH;
-		$this->thumbHeight = self::IMG_HEIGHT;
-		$this->thumbRows = self::THUMBS_NUM_ROWS;
-		$this->numThumbsPerRow = self::THUMBS_PER_ROW;
+		$this->thumbWidth = static::IMG_WIDTH;
+		$this->thumbHeight = static::IMG_HEIGHT;
+		$this->thumbRows = static::THUMBS_NUM_ROWS;
+		$this->numThumbsPerRow = static::THUMBS_PER_ROW;
 		$this->trendingArticles = $trendingArticles;
 		$this->mainPageUrl = Title::newMainPage()->getFullUrl();
-		//$this->larrSrc = $this->wg->ExtensionsPath."/wikia/AppPromoLanding/images/arrow-left-long.svg";
 		$this->larrSvgCode = "<svg width=\"22px\" height=\"16px\" viewBox=\"0 0 22 16\" version=\"1.1\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">
 								<title>BB56E3FE-7480-48C0-96B3-848DAFB20649</title>
 								<desc>Created with sketchtool.</desc>
```

Comment:
That tool looks really cool but doesn't appear to be able to handle this SVG... the optimized result doesn't render anything. I trimmed off some obvious cruft though & made the 'title' better.